server/graphql/v2/query/collection/OrdersCollectionQuery.ts

```diff
@@ -173,7 +173,7 @@ export const OrdersCollectionResolver = async (args, req: express.Request) => {
     const paymentMethod = await fetchPaymentMethodWithReference(args.paymentMethod, {
       sequelizeOpts: { attributes: ['id'], include: [{ model: models.Collective }] },
     });
-    if (!req.remoteUser?.isAdminOfCollective(paymentMethod.Collective)) {
+    if (!req.remoteUser?.isAdminOfCollective((paymentMethod as any).Collective)) {
```

Comment:
We should rather have `Collective` as an optional attribute of payment method, like we do in https://github.com/opencollective/opencollective-api/blob/3117bff193fa20c2a4f5476e6a290e8d3cb79e08/server/models/Expense.ts#L79 for example